src/app/Dashboard/Dashboard.tsx

```diff
@@ -270,26 +310,33 @@ export const Dashboard: React.FC<DashboardProps> = (_) => {
 
   return (
     <TargetView pageTitle="Dashboard" compactSelect={false}>
-      <Grid id={'dashboard-grid'} hasGutter>
-        {cardConfigs.map((cfg, idx) => (
-          <GridItem span={cfg.span} key={cfg.id} order={{ default: idx.toString() }}>
-            {React.createElement(getConfigByName(cfg.name).component, {
-              ...cfg.props,
-              dashboardId: idx,
-              actions: [
-                <DashboardCardActionMenu
-                  key={`${cfg.name}-actions`}
-                  onRemove={() => handleRemove(idx)}
-                  onResetSize={() => handleResetSize(idx)}
-                />,
-              ],
-            })}
+      <ChartContext.Provider value={{ controller: chartController.current }}>
+        <Grid id={'dashboard-grid'} hasGutter>
+          {cardConfigs.map((cfg, idx) => (
+            <GridItem span={cfg.span} key={cfg.id} order={{ default: idx.toString() }}>
+              {hasConfigByName(cfg.name) ? (
```

Comment:
Maybe this check should be applied the outer GridItem since this will add an empty GridItem that has some padding causing larger gaps :D

Reply:
This comes from messing around with local storage.

![Screenshot from 2023-02-09 00-33-14](https://user-images.githubusercontent.com/68053619/217726792-2eccb711-6868-42d4-9832-30104184eda4.png)